docs/UserGuide.adoc

```diff
@@ -284,6 +304,8 @@ e.g. `edit 2 n/James Lee e/jameslee@example.com`
 e.g. `removegroup g/publicity`
 * *Find* : `find KEYWORD [MORE_KEYWORDS]` +
 e.g. `find James Jacob`
+* *Find* : `findby FIELD KEYWORD [MORE_KEYWORDS]` +
```

Comment:
Suggestions for minor improvements to be made in a future PR:
Change the title from `Find`

Reply:
As discussed in class, let's just merge the two commands